napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -383,6 +397,14 @@ def setup_ui(self, enabled=True):
         self.plugin_name.setFont(font15)
         self.row1.addWidget(self.plugin_name)
 
+        self.npe2_icon = QLabel(self)
+        self.npe2_text = QLabel(self)
+        self.npe2_text.setText('npe2')
+        self.npe2_text.hide()
+        self.row1.addWidget(self.npe2_icon)
+        self.row1.addWidget(self.npe2_text)
+
```

Comment:
@tlambert03  I made the function above, but I don't know if I can add this icon Qlabel code to it easily.  It would be inserting these QLabels after the whole setup is created.  You can probably do it, but it seems messy.

Reply:
yep, just use `insertWidget`:

```py
    def _handle_npe2_plugin(self):
        npe2_icon = QLabel(self)
        icon = QColoredSVGIcon.from_resources('logo_silhouette')
        npe2_icon.setPixmap(icon.colored(color='#33F0FF').pixmap(20, 20))
        self.row1.insertWidget(2, QLabel('npe2'))
        self.row1.insertWidget(2, npe2_icon)
        self.enabled_checkbox.setEnabled(False)
        self.enabled_checkbox.setToolTip(
            'This is a npe2 plugin and cannot be enabled/disabled at this time.'
        )
```

Reply:
ok...well, that's not messy. 😆  thank you!